.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
+}
```

Comment:
Now VSCode users will enjoy linting on the fly (provided they have the ESLint plugin installed)